run.go

```diff
@@ -153,12 +153,6 @@ func (b *Builder) Run(command []string, options RunOptions) error {
 	}()
 	g := generate.New()
 
-	if b.OS() != "" {
```

Comment:
Just curious, was this changed for the spec update or just something you meant to change?